aws-spi-pekko-http/src/main/scala/org/apache/pekko/stream/connectors/awsspi/PekkoHttpClient.scala

```diff
@@ -185,14 +194,20 @@ object PekkoHttpClient {
         connectionPoolSettings.getOrElse(ConnectionPoolSettings(as)),
         resolvedOptions
       )
+
+      val connectionContext =
+        if (resolvedOptions.get(SdkHttpConfigurationOption.TRUST_ALL_CERTIFICATES).booleanValue())
```

Comment:
I had to add support for `TRUST_ALL_CERTIFICATES` option so the test suite can connect to an insecure server. Unfortunately, skipping cert validation is not so easy (see for example https://github.com/akka/akka-http/issues/4042)